bot-plutus-interface.cabal

```diff
@@ -24,14 +24,12 @@ source-repository head
 
 common common-lang
   ghc-options:
-    -Wall -Wcompat -Wincomplete-record-updates
-    -Wincomplete-uni-patterns -Wredundant-constraints -fobject-code
-    -fno-ignore-interface-pragmas -fno-omit-interface-pragmas
-    -fplugin=RecordDotPreprocessor -Werror
+    -Wall -Wcompat -Wincomplete-uni-patterns -Wredundant-constraints
```

Comment:
Is dropping -Wincomplete-record-updates not dangerous?

Reply:
I don't think it's dangerous, but it will make readability a bit a worse.
However, as long as haskell.nix has this bug, I don't see any other way around it (I think upgrading to ghc9 is still impossible because of cardano-node)

Reply:
Sad, oh well